sdk/core/azure-core-amqp/src/amqp/private/message_sender_impl.hpp

```diff
@@ -84,6 +86,12 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
         Models::_internal::AmqpError>
         m_sendCompleteQueue;
 
+#if SENDER_SYNCHRONOUS_CLOSE
```

Comment:
This is the equivalent of #IF FALSE given the above define.  Why do we want this code around if its always not compiled?

Reply:
Because it's code I neutered in an attempt to get the CI pipeline to pass.

I RELLY don't want to lose this code - it's 100% correct and works just fine on Windows. But on Linux it fails miserably in uAMQP (that's why the tests are disabled on linux).

But I can't get further in diagnosing the problem without having access to the uAMQP codebase to add diagnostics. So it needed to be tabled in favor of vendoring uAMQP.